static/elements/chromedash-upcoming-milestone-card.js

```diff
@@ -0,0 +1,170 @@
+import {LitElement, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import style from '../css/elements/chromedash-upcoming-milestone-card.css';
+
+class ChromedashUpcomingMilestoneCard extends LitElement {
+  static styles = style;
+  static get properties() {
+    return {
+      // Assigned in schedule-apge.js, value from Django
+      showBlink: {type: Boolean},
+      starredFeatures: {type: Array},
+      templateContent: {type: Object},
+      channel: {type: Object},
+      showDates: {type: Boolean},
+      removedStatus: {type: Array},
+      deprecatedStatus: {type: Array},
+      signedIn: {type: Boolean},
+    };
+  }
+
+  /**
+   *  Returns the number of days between a and b.
+   *  @param {!Date} a
+   *  @param {!Date} b
+   *  @return {!{days: number, future: boolean}}
+   */
+  _dateDiffInDays(a, b) {
+    // Discard time and time-zone information.
+    const utc1 = Date.UTC(a.getFullYear(), a.getMonth(), a.getDate());
+    const utc2 = Date.UTC(b.getFullYear(), b.getMonth(), b.getDate());
+
+    const MS_PER_DAY = 1000 * 60 * 60 * 24;
+    const daysDiff = Math.floor((utc2 - utc1) / MS_PER_DAY);
+    return {days: Math.abs(daysDiff), future: daysDiff < 1};
+  }
+
+  _computeDate(dateStr) {
```

Comment:
Is this computing a date or just formatting it differently?

Reply:
This function was present in the OLD schedule page. I have not added anything to it. It seems to convert date string obtained from API to a date object so that it is easier to perform date operations like subtracting two dates.

Reply:
OK.  I think the name might be more clear with some other verb, but I don't feel strongly about it.